packages/loaders/src/styled/StyledSVG.ts

```diff
@@ -9,13 +9,13 @@ import styled, { css, DefaultTheme, ThemeProps } from 'styled-components';
 import { getColor, retrieveComponentStyles } from '@zendeskgarden/react-theming';
 
 interface IStyledSVGProps {
-  dataGardenId: string;
+  $dataGardenId: string;
   color?: string;
   fontSize?: string | number;
   width: number | string;
   height: number | string;
```

Comment:
```suggestion
  $color?: string;
  $fontSize?: string | number;
  $width: number | string;
  $height: number | string;
```

We can remove these from the DOM since they only need to inform the CSS